integration-tests/src/test/resources/features/bridge.feature

```diff
@@ -12,7 +12,5 @@ Feature: Bridge tests
     Then the Bridge "mybridge" is not existing within 2 minutes
     And the Ingress of Bridge "mybridge" is not available within 1 minute
 
-    And the Manager metric 'manager_bridge_status_change_total{status="PREPARING",}' count is at least 1
-    And the Manager metric 'manager_bridge_status_change_total{status="PROVISIONING",}' count is at least 1
-    And the Manager metric 'manager_bridge_status_change_total{status="READY",}' count is at least 1
-    And the Manager metric 'manager_bridge_status_change_total{status="DELETED",}' count is at least 1
+    And the Manager metric 'managed_services_api_rhose_operation_success_count_total{operation="provision",resource="bridge",}' count is at least 1
+    And the Manager metric 'managed_services_api_rhose_operation_success_count_total{operation="delete",resource="bridge",}' count is at least 1
```

Comment:
Can you please add here at least also:
```
    And the Manager metric 'managed_services_api_rhose_operation_count_total{operation="provision",resource="bridge",}' count is at least 1
    And the Manager metric 'managed_services_api_rhose_operation_count_total{operation="delete",resource="bridge",}' count is at least 1
```
So we have at least these metrics visibly covered.

Reply:
Sure thing.